src/components/organisms/Navbar/NavbarLink/NavbarLink.tsx

```diff
@@ -36,36 +36,62 @@ export const navbarLinkStyles = css<StyledNavbarLinkProps>`
   display: inline-flex;
   padding: ${spacing[3]} ${spacing[4]} ${spacing[4]};
 
+ 
+
   &:active,
   &:hover {
     color: ${colors.actionDark};
     opacity: ${({ active }) => (active ? 1 : 0.88)};
   }
+  
 
   ${maxEqualToMedia.desktop`
     ${({ withChevron }: StyledNavbarLinkProps) =>
       withChevron &&
-      css`
+      `
         color: ${colors.greyDarkest};
         font-size: ${typography.sizes.text.small};
         font-weight: ${typography.weights.bold};
         text-transform: uppercase;
       `}
   `}
-
+  
   ${minMedia.desktop`
-    ${({ isDropdownLink }: StyledNavbarLinkProps) =>
+    ${({ isDropdownLink, isDropdownHeading }: StyledNavbarLinkProps) =>
       isDropdownLink &&
+      !isDropdownHeading &&
       css`
-        padding: ${spacing[3]};
+        padding: ${spacing[3]} ${spacing[2]};
+        width: calc(100% - 16px);
+        margin-left: 8px;
+        margin-right: 8px;
```

Comment:
I think both spacing variable and hardcoded `px` values are fine here. It would be nice to keep it consistent though - use one approach or the other :)